pkgs/applications/blockchains/charge-lnd/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, fetchFromGitHub, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "charge-lnd";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "accumulator";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1m1ic69aj2vlnjlp4ckan8n67r01nfysvq4w6nny32wjkr0zvphr";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    aiorpcx
+    colorama
+    googleapis-common-protos
+    grpcio
+    protobuf
+    six
+    termcolor
+  ];
+
+  postInstall = ''
+    mkdir -p $out/share/doc
+    cp README.md charge.config.example $out/share/doc
```

```suggestion
    mkdir -p $out/share/doc/charge-lnd
    cp README.md charge.config.example $out/share/doc/charge-lnd
```

Otherwise we get problems when merging the package in a user environment.

---

Ah right, didn't realize this, fixed.